lib/Cake/Utility/Inflector.php

```diff
@@ -495,7 +495,12 @@ public static function underscore($camelCasedWord) {
  */
 	public static function humanize($lowerCaseAndUnderscoredWord) {
 		if (!($result = self::_cache(__FUNCTION__, $lowerCaseAndUnderscoredWord))) {
-			$result = ucwords(str_replace('_', ' ', $lowerCaseAndUnderscoredWord));
+			$result = str_replace('_', ' ', $lowerCaseAndUnderscoredWord);
+			if (function_exists('mb_convert_case') && Multibyte::checkMultibyte($result)) {
```

Comment:
I tried check exists mb_convert_case only, but many test case broken.

mb_convert_case results not  equal ucwords.
eg.
`mb_convert_case('contact addForm', MB_CASE_TITLE)` = 'Contact Addform'
`ucwords('contact addForm');` = 'Contact AddForm'


I seems to be better, if a string that contains a multibyte,  it does not convert to title case.